go.mod

```diff
@@ -5,30 +5,31 @@ go 1.17
 require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/go-logr/logr v0.4.0
-	github.com/go-logr/zapr v0.2.0 // indirect
+	github.com/go-logr/zapr v0.4.0 // indirect
 	github.com/google/uuid v1.1.2
 	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.14.0
+	github.com/onsi/gomega v1.15.0
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
-	github.com/talos-systems/capi-utils v0.0.0-20210917140904-9587089e8425
-	github.com/talos-systems/cluster-api-bootstrap-provider-talos v0.3.0
+	github.com/talos-systems/capi-utils v0.0.0-20210930173123-b2f8f83d3df6
+	github.com/talos-systems/cluster-api-bootstrap-provider-talos v0.3.0-alpha.1.0.20210928191314-8c7fec8e373b
```

Comment:
should be 0.4.0-alpha.0 if we push the tag?

Reply:
yep, good point